libs/utils/executor.py

```diff
@@ -272,13 +272,29 @@ def _setup_group(self, tc, group):
         # Configure each required attribute
         group.set(**tc['cgroups']['conf'][kind][name])
 
+    def _setup_files(self, tc):
+        errors = False
+        if 'files' not in tc:
+            logging.debug('%14s - Configuration not provided', 'Files')
+            return True
+        for name in tc['files']:
+            logging.info('%14s - Write value into file: \'%s\' -> \'%s\'',
+                         'Files', tc['files'][name], name)
+            try:
+                self.target.write_value(name, tc['files'][name], True)
```

Comment:
Forcing the value check can fails for files like sched_features.

In devlib a prefix token is used to mark files for which we do not want to check the value, i.e. append-mode files.

Maybe we can use something like:

```python
check_value = True
tc['files'][name].startswith('+/'):
    check_value = False
self.target.write_value(name, tc['files'][name], check_value)
```

which should allow to have something like:

```json
    "files" : {
        "+/sys/kernel/debug/sched_features" : "ENERGY_AWARE"
    }
```